Atomic/util.py

```diff
@@ -794,6 +795,54 @@ def load_scan_result_file(file_name):
     """
     return json.loads(open(os.path.join(file_name), "r").read())
 
+
+class InstallData(object):
+
+    @staticmethod
+    def read_install_data():
+        if os.path.exists(ATOMIC_INSTALL_JSON):
+            return load_scan_result_file(ATOMIC_INSTALL_JSON)
+        return {}
+
+    @staticmethod
+    def write_install_data(install_data):
+        with open(ATOMIC_INSTALL_JSON, 'w') as write_data:
+            json.dump(install_data, write_data)
```

Comment:
Should we use file locks?  To make sure that two processes accessing the file at the same time don't overight?  Or should we create a different file for each installed image?